inc/class-cachify-redis.php

```diff
@@ -115,8 +115,21 @@ public static function clear_cache() {
 			return;
 		}
 
-		/* Flush */
-		@self::$_redis->flushAll();
+		/* Delete all cache entries for this site */
+		$host = wp_unslash( $_SERVER['HTTP_HOST'] );
+		$keys = self::$_redis->keys( $host . '*' );
```

Comment:
`keys()` returns the keys with the configured prefix, but `del()` or `unlink()` do not expect it and will add it again before sending the command to Redis.

This quirk of phpredis was discussed in https://github.com/phpredis/phpredis/issues/1117#issuecomment-283159870.